### ISSUE TITLE
Unable to attach R session

### ISSUE THREAD

**Issue by fwten** [Ikuyadeu/vscode-R#355]
Hello, excellent work you're doing here!

I'm having issues attaching the R session, and would appreciate it if you could provide some assistance or guidance here :) 

**Describe the bug**
R session can not be attached by clicking `R: (not attached)`

**To Reproduce**
1. Run `R` or `radian` from Terminal pane
2. Click on `R: (not attached)` 
3. radian outputs: ```getOption('vscodeR')$attach()
Error: attempt to apply non-function```
4. `getOption('vscodeR')` returns `NULL`

(If related) settings.json
```
{
    "r.sessionWatcher": true,
    "r.bracketedPaste": true,
    "r.alwaysUseActiveTerminal": true,
    "r.lsp.diagnostics": false,
    "r.rterm.option": [],
}
```

**Environment (please complete the following information):**
 - OS: macOS 10.14.6, SSH-ing into Ubuntu 18.04
 - VSCode Version: 1.46.0 (on macOS)
 - R Version: 3.6.3 (on Ubuntu)
 - vscode-R version: 1.4.1 (on macOS and Ubuntu 18.04)

**Additional context**
If it helps, this is my `.Rprofile` on the Ubuntu server:
```
Sys.setenv(TERM_PROGRAM = "vscode")
source(file.path(Sys.getenv(
  if (.Platform$OS.type == "windows") "USERPROFILE" else "HOME"),
  ".vscode-R", "init.R")
)
```
Also, not sure if it's related to the session but, currently all my plots are displayed in a new X11 window (through XQuartz). Is there a way to make it appear in a pane inside VS Code instead?

Thanks for looking into this and let me know if you need me to perform some tests or provide more information.

**Comment by renkun-ken** [comment]
Did session watcher work for you, or it stop working after upgrading to vscode-R 1.4.1 from 1.4.0?

From the error message

```
getOption('vscodeR')$attach() Error: attempt to apply non-function
```

It indicates that `.Rprofile` is not sourced at all. Would you like to run the following code in your R session manually and see what happens?

```r
Sys.setenv(TERM_PROGRAM = "vscode")
source(file.path(Sys.getenv(
  if (.Platform$OS.type == "windows") "USERPROFILE" else "HOME"),
  ".vscode-R", "init.R")
)
```

**Comment by fwten** [comment]
Actually, I started trying it from 1.4.1, so I can't comment on 1.4.0.

But you're right, I ran the code in my R session manually, and the R session is now attached, and the plots are also showing up in a pane as well, so thank you!

Now that the plots are rendered in VS Code, is there a way to trigger the re-rendering of the figure when you resize the pane? This behaviour is present in both X11 and RStudio plots.

**Comment by renkun-ken** [comment]
>But you're right, I ran the code in my R session manually, and the R session is now attached, and the plots are also showing up in a pane as well, so thank you! How could I fix this so that I don't have to run the code manually every time?

Did you put the code in `~/.Rprofile` or workspace `.Rprofile`? We have to figure out why your `.Rprofile` does not work and session watcher will work if we resolve this.

>Now that the plots are rendered in VS Code, is there a way to trigger the re-rendering of the figure when you resize the pane? This behaviour is present in both X11 and RStudio plots.

We only provide very simple mechanism of viewing plots. If you need more features such as redraw on resize and high quality graphics, I strongly recommend that you use the newly developed https://github.com/nx10/httpgd.

```r
httpgd::httpgd()
utils::browseURL(httpgd::httpgdURL())
```

Currently, we have not figured out how we could make it work more seamlessly with vscode-R but it works very nicely already.

**Comment by fwten** [comment]
I put the code in `~/.Rprofile`, and I did not have a `.Rprofile` in my workspace. Interestingly, session watcher works now after I added a newline at the end of the `~/.Rprofile` file, and this was brought to my attention when I was installing the `httpgd` package you recommended.

I've tried this after restarting my VS Code several times and changing Workspace, so I think the issue can be considered as solved!

> We only provide very simple mechanism of viewing plots. If you need more features such as redraw on resize and high quality graphics, I strongly recommend that you use the newly developed https://github.com/nx10/httpgd.

This is indeed very amazing! Once again, thank you very much :+1:
